### PR TITLE
Create jrsh formula

### PR DESCRIPTION
**Pull request by Krasnyanskiy**

### CODE REVIEW

**Comment by mistydemeo** [comment]
Passing on this for now; this doesn't look like it currently meets our [notability criteria](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#niche-or-self-submitted-stuff).

I'd suggest maintaining this formula in a tap of your own, which will still provide users with a convenient way to install it using Homebrew.

Thanks!

**Comment by Krasnyanskiy** [comment]
@michaelpq So other users will not be able to install it via `brew install jrsh`?

**Comment by mistydemeo** [comment]
No, but users would be able to `brew install Krasnyanskiy/jrsh/jrsh`.

**Comment by Krasnyanskiy** [comment]
Ok. And where can I find some docs to get info about tap installation? Do you have some links?

**Comment by bfontaine** [comment]
@Krasnyanskiy There are [some docs here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/brew-tap.md).

TL;DR:

1. Create a GitHub repository `homebrew-mytap` (replace `mytap` with what you want)
2. Put your `jrsh.rb` in it
3. Done. Users can now install your formula with `brew install Krasnyanskiy/mytap/jrsh`.

**Comment by Krasnyanskiy** [comment]
Ok, thanks. I will try to do it.
